metal/v1/test/api_self_service_reservations_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+Metal API
+
+Testing SelfServiceReservationsApiService
+
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech);
+
+package v1
+
+import (
+	"context"
+	"testing"
+
+	openapiclient "github.com/equinix-labs/metal-go/metal/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_v1_SelfServiceReservationsApiService(t *testing.T) {
+	configuration := openapiclient.NewConfiguration()
+	apiClient := openapiclient.NewAPIClient(configuration)
+
+	t.Run("Test SelfServiceReservationsApiService CreateSelfServiceReservation", func(t *testing.T) {
+		t.Skip("skip test") // remove to run test
+
+		var projectId string
+
+		resp, httpRes, err := apiClient.SelfServiceReservationsApi.CreateSelfServiceReservation(context.Background(), projectId).Execute()
+
+		require.Nil(t, err)
+		require.NotNil(t, resp)
+		assert.Equal(t, 200, httpRes.StatusCode)
+	})
+
+	t.Run("Test SelfServiceReservationsApiService FindSelfServiceReservation", func(t *testing.T) {
+		t.Skip("skip test") // remove to run test
```

Comment:
The tests generated by `openapi-generator` are all explicitly skipped, and there is no way to tell the generator to enable the tests.  If this gets merged as-is, I'll add an issue to enable at least one test; we'll have quite a bit to figure out for that, since the generated tests don't include mocking to run as unit tests and (naturally) don't have the necessary setup to run as end-to-end tests.

Reply:
> don't have the necessary setup to run as end-to-end tests.

I see. There is no mocking and the `Execute()` calls do not prepare meaningful request bodies.  We would encounter many undefined required parameters, as-is.